lib/travis/build/script/r.rb

```diff
@@ -0,0 +1,368 @@
+module Travis
+  module Build
+    class Script
+      class R < Script
+        DEFAULTS = {
+          # Basic config options
+          cran: 'http://cran.rstudio.com',
+          warnings_are_errors: false,
```

Comment:
Please do not make `warning` equal `error`. There _are_ false positives in `R CMD check`.  This would make it useless.  IIRC correctly I have at least one package which _permanently_ exhibits one warning.  Wrongly.

On the margin I think I'd fork, or stick to r-travis which I can modify.  

If you must, make it an option that defaults to `FALSE`.